packages/circuit-ui/components/ToastContext/ToastContext.tsx

```diff
@@ -24,7 +24,7 @@ import { css } from '@emotion/react';
 
 import { useStack, StackItem } from '../../hooks/useStack';
 import styled, { StyleProps } from '../../styles/styled';
-import { spacing } from '../..';
+import { spacing } from '../../styles/style-mixins';
```

Comment:
This import from the package root had been causing issues for ages. I'm glad I finally found it.